package.json

```diff
@@ -59,6 +59,7 @@
         "@vitejs/plugin-legacy": "^1.5.0",
         "@vitejs/plugin-vue": "^1.3.0",
         "@vue/compiler-sfc": "^3.1.5",
+        "babel-eslint": "^10.1.0",
```

Comment:
Are we using the right package? This package has been deprecated according to https://www.npmjs.com/package/babel-eslint.

Reply:
Switched to new babel-eslint-parser. I guess I won't pull in babel, since we aren't using experimental syntax (?)